pkg/kubelet/eviction/helpers.go

```diff
@@ -91,6 +90,16 @@ func init() {
 	signalToResource[evictionapi.SignalPIDAvailable] = resourcePids
 }
 
+func getPodPriority(pod *v1.Pod) int32 {
```

Comment:
we now have two implementations of this, I think we should have a single implementation under pkg/api/pod/util.go

Reply:
This was discussed in a previous (closed) PR, but for something small like this, having two copies is not the end of the world. I support moving it to a neutral library (neither kubelete nor scheduler) but I don't think that should be part of this PR.

Reply:
Not doing it in this PR is fine, but we should move it eventually, even small things like this can diverge in semantics without noticing.

Reply:
Or add a todo